<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LazyFixTool</class>
 <widget class="QDialog" name="LazyFixTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>606</width>
    <height>622</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Fix lazily-made masked GIFs</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../_resources/maintainer.qrc">
    <normaloff>:/maintainer.ico</normaloff>:/maintainer.ico</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,0,0,0,10000">
   <item row="2" column="0" colspan="4">
    <widget class="QTabWidget" name="whatToConvert">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="filesTab">
      <attribute name="title">
       <string>Process files</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QLabel" name="filesDropArea">
         <property name="acceptDrops">
          <bool>true</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">border-color: rgb(46, 52, 54);
border-style: dotted;
border-width: 2px;
color: black;
border-radius: 15px;</string>
         </property>
         <property name="text">
          <string>Drop your masked GIFs files into this window for a quick porcessing</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="episode">
      <attribute name="title">
       <string>Batch processing for level/episode</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Input directory (for example, an episode root or level custom folder):</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QCheckBox" name="flagRemoveOld">
         <property name="text">
          <string>Don't create backup for old images</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="inPathBrowse">
         <property name="text">
          <string>Browse...</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLineEdit" name="inPath">
         <property name="placeholderText">
          <string>Path to an input directory with a source data</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Flags</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QCheckBox" name="flagScanSubdirs">
         <property name="text">
          <string>Process all content of subdirectories</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>What do you want?</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="3">
    <widget class="QPushButton" name="doMadJob">
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="4">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>This wizard will help you to automatically fix any SMBX 1.3 graphics (in a GIF format with the use of bitmask) that have a bad-designed invalid mask (e.g. made with a lazy effort). Masks made with a lazy effort are masks made by the copying of a front image and filling the background with white. This making method for masks gives an illusion of quick work, however, these masks are invalid and may cause strange glitches on some graphical engines and converters. After processing these sprites, all invalid masks will be turned into the valid form of the black shape as they must be, and therefore, they can be freely used everywhere without glitches.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="4">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Output log</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QTextBrowser" name="outputLog">
        <property name="font">
         <font>
          <family>Monospace</family>
          <pointsize>8</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="closeBox">
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../_resources/maintainer.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>closeBox</sender>
   <signal>clicked()</signal>
   <receiver>LazyFixTool</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>318</x>
     <y>302</y>
    </hint>
    <hint type="destinationlabel">
     <x>238</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
